### PR TITLE
fixes a small 'thinko'

### PR DESCRIPTION
**Pull request by eddelbuettel**
only whitespace in newDateVector
rolls minor version
updates debian/changelog

### CODE REVIEW

**Comment by thirdwing** [comment]
Let's merge this small PR first?

**Comment by eddelbuettel** [comment]
No worries -- I don't really care about the order.  In a way, @coatless was first, and @jimhester has a more important fix. 

This particular one is my bad, and I noticed when trying to update something in the [RcppExamples](https://github.com/eddelbuettel/rcppexamples) package.  This is not urgent as it depends on the [toggle in Rcpp.h being defined](https://github.com/RcppCore/Rcpp/blob/master/inst/include/Rcpp.h#L59) so it is not yet biting anyone unless that is one (or someone turns it one) and they try `operator+` on a new `DatetimeVector`.  

So in a nutshell it doesn't really matter but if you want it merged first go ahead.  Else I'll get to in the new few days.
